src/Datasource/ConnectionManager.php

```diff
@@ -150,20 +150,20 @@ public static function parseDsn(string $config): array
      * ConnectionManager::alias('test_things', 'things');
      * ```
      *
-     * @param string $alias The alias to add. Fetching $source will return $alias when loaded with get.
-     * @param string $source The connection to add an alias to.
+     * @param string $source The existing connection to alias.
+     * @param string $alias The alias name that resolves to `$source`.
      * @return void
      * @throws \Cake\Datasource\Exception\MissingDatasourceConfigException When aliasing a
      * connection that does not exist.
      */
-    public static function alias(string $alias, string $source): void
+    public static function alias(string $source, string $alias): void
```

Comment:
Doesn't this invert the behavior as well?

Reply:
No, I changed this line `static::$_aliasMap[$alias] = $source;`